src/components/file/File.js

```diff
@@ -783,6 +783,42 @@ export default class FileComponent extends BaseComponent {
         this.uploadStatusList.appendChild(uploadStatus);
 
         if (fileUpload.status !== 'error') {
+          // Track uploads in progress.
+          if (fileService.uploadsInProgress === undefined) {
+            const cssClass = 'uploads-in-progress';
+            var submitButton = this.root.element
+                .querySelector('.formio-component-submit')
```

Comment:
@franke-mc @travist This button does not exist in Wizard forms and giving null for next querySelector so file uploading is not working in wizard forms.

Reply:
Sorry for that. Could you please try https://github.com/formio/formio.js/pull/1309 ?